spikesorters/ironclust/shellscript.py

```diff
@@ -0,0 +1,167 @@
+import subprocess
+import tempfile
+import shutil
+import signal
+import os
+import time
+from typing import Optional, List, Any
+
+
+class ShellScript():
```

Comment:
This could be done on sorter_tools.py and share by other sorter no ?

Reply:
yes, but I don't think it's solid yet. We wanted to make __enter__ and __exit__ stuff, and improve some other things.

Reply:
let's keep it here for now. We can move it to the sortertools later. All matlab and cli sorters chould use this IMO